src/thunderbots/software/ai/passing/evaluation.cpp

```diff
@@ -0,0 +1,52 @@
+// TODO: start of file comment
+
+#include "ai/passing/evaluation.h"
+#include "util/parameter/dynamic_parameters.h"
+
+double getStaticPositionQuality(Field field, Point position) {
+
+    double positionQuality = 1;
+    double length          = field.length() / 2;
```

Comment:
Consider naming these variables `half_field_length` and `half_field_width` for clarity?

Reply:
Changed